website/docs/cloud-docs/api-docs/index.mdx

```diff
@@ -129,7 +130,9 @@ The following entitlements are available:
 
 [run tasks]: /cloud-docs/api-docs/run-tasks
 
-[Sentinel]: /cloud-docs/sentinel
+[Sentinel]: /cloud-docs/policy-enforcement#sentinel
```

Comment:
Will probably need a redirect put in for this.  @laurapacilio can help with that